uwrt_mars_rover_utils/include/uwrt_mars_rover_utils/uwrt_can.h

```diff
@@ -131,6 +138,59 @@ class UWRTCANWrapper {
 
     return bytes_sent == sizeof(struct can_frame);
   }
+
+  template <class T>
+  bool writeToIDwithAck(T data, uint32_t id) {
+    // check socket has been initialized
+    if (!initialized_) {
+      ROS_ERROR_STREAM("Please intialize CAN wrapper before using it");
```

Comment:
I added a getLoggerName in #124 that I think you should use here. I think all Ros console calls should be named

Reply:
#124 has merged.  You can use the function